0_intro/stage/exos_python.ipynb

```diff
@@ -29,6 +29,37 @@
     "5. Écrire une fonction vérifiant que tous les éléments d'une liste sont pairs."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        return ([ l[-i] for i in range(1,len(l)+1) ])
</pre><p>Parenthèses pas obligatoires</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>